scraper/scraper_utils.py

```diff
@@ -0,0 +1,116 @@
+from random import randint
+import requests
+import csv
+from typing import List, Dict
+import pickle
+import uuid
+
+SEARCHED_FIELDS_OTD = {
+    'offer_type': '"user_type":', 
+    'floor': '"Floor_no":', 
+    'area': '"Area":',
+    'rooms': '"Rooms_num":', 
+    'offer_type_of_building': 'ProperType":', 
+    'market': '"MarketType":', 
+    'longtitude': '"long":', 
+    'latitude': '"lat":', 
+    'price': '"Price":'
+}
+
+SEARCHED_FIELDS_OX = {
+    'offer_type': ['"user_type":', 3],
+    'floor': ['"Poziom', 3], 
+    'area': ['"Powierzchnia', 3],
+    'rooms': ['"rooms', 3], 
+    'offer_type_of_building': ['"Rodzaj zabudowy', 3], 
+    'market': ['"Rynek', 3], 
+    'longtitude': ['"lon":', 0], 
+    'latitude': ['"lat":', 0], 
+    'price': ['"price"', 3]
+}
+
+def get_random_number_from_range(left_boundary: int, right_boundary: int) -> int:
+    return randint(left_boundary, right_boundary)
+
+def get_flat_ox_data(link: str) -> dict:
+    request = requests.get(link)
+    str_request_data = str(request.content).replace('\\', '').split(',')
+    filtered_data = {}
+
+    try:
+        for key, value in SEARCHED_FIELDS_OX.items():
+            for i, item in enumerate(str_request_data):
+                if value[0] in item:
+                    filtered_data[key] = str_request_data[i+value[1]]
+
+        for item in filtered_data.keys():
+            if 'floor' in item:
+                filtered_data['floor'] = filtered_data['floor'].split(':')[1].strip('"}flor_')
```

Comment:
misspell or is it really 'flor_' XD?

Reply:
it strips schars, so no need to type other o xD

Reply:
I would say either we parse it on scraper side or pass everything unparsed in csv.
Since **artifact.csv** is a csv, some unscraped strings may contain comma and then corrupt file